src/api/app/models/issue_tracker.rb

```diff
@@ -52,6 +52,9 @@ def valid_issue_name?(name)
   def show_url_for(issue, html = nil)
     return unless issue
     url = show_url.gsub('@@@', issue)
+    # generic github tracker needs special handling since they inject '/issues/' inside
+    # github handles issue/pull redirection on it's own luckily
+    url.gsub!(/\#/, '/issues/') if kind == 'github'
```

Comment:
Why need this to be a reg exp? :confused:

Reply:
Changed to `'#'`